lib/ui/window.dart

```diff
@@ -67,6 +98,12 @@ class FlutterView {
     return platformDispatcher._viewConfigurations[viewId]!;
   }
 
+  /// The [Display] this view is drawn in.
+  Display get display {
+    assert(platformDispatcher._displays.containsKey(_viewConfiguration.displayId));
+    return platformDispatcher._displays[_viewConfiguration.displayId]!;
+  }
+
   /// The number of device pixels for each logical pixel for the screen this
```

Comment:
There is a little bit of duplicated information available now as dpr as also available via the display getter now. It's probably not an issue, but a little strange? Presumably, both will always have the same value?

Reply:
DPR really belongs on the display. I'm half tempted to deprecate it on View, but that seems like it'd create a lot of unnecessary churn.

But yes, DPR on the view will always match DPR from the display.

Reply:
You might want to add to the dart doc to DPR on view that says window is the prefered way if you indeed think that is the case.

Reply:
You need the DPR if you want to figure out logical pixel sizes. I'll document that they're the same.

Reply:
Added some docs.